packages/corvid-e2e-tests/.eslintrc

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
we no longer have local eslint config files
you can just delete this file, assuming the global one fits